modules/installation-overview.adoc

```diff
@@ -7,50 +7,148 @@
 [id="installation-overview-{context}"]
 = {product-title} installation overview
 
-You use the {product-title} installer to create {product-title} clusters in
-//a variety of environments. The same installer allows you to install
-//{product-title} on your own infrastructure or in
+The {product-title} 4 installation program offers you flexibility. You can use the installation
```

Comment:
4 -> `{product-version}` or `{product-major-version}` or some such (here and below)?